Makefile

```diff
@@ -28,7 +28,7 @@ ifdef DEBUG
   CCFLAGS += -ggdb -O0
   LDFLAGS += -ggdb
 else
-  CCFLAGS += -Os
+  CCFLAGS += -O2 
```

Comment:
It turns out that I can't find the option that does the alignment of strings. There are only three reported differences (controlled by -f flags) between -Os and -O2. One of these is -finline-functions that we already override to off. The other two looked really obscure.